
package gen;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.5.4
 * 2023-03-29T10:28:16.037+03:00
 * Generated source version: 3.5.4
 */

@WebFault(name = "quadraticEducationException", targetNamespace = "http://math.ws.divanov")
public class QuadraticEducationException extends Exception {

    private gen.ErrorResponse faultInfo;

    public QuadraticEducationException() {
        super();
    }

    public QuadraticEducationException(String message) {
        super(message);
    }

    public QuadraticEducationException(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public QuadraticEducationException(String message, gen.ErrorResponse quadraticEducationException) {
        super(message);
        this.faultInfo = quadraticEducationException;
    }

    public QuadraticEducationException(String message, gen.ErrorResponse quadraticEducationException, java.lang.Throwable cause) {
        super(message, cause);
        this.faultInfo = quadraticEducationException;
    }

    public gen.ErrorResponse getFaultInfo() {
        return this.faultInfo;
    }
}
